protocol/internal/testdriver/main.go

```diff
@@ -0,0 +1,15 @@
+package main
```

Comment:
`testdriver` package is used only in tests, this `main.go` then can be used (with a simple replacement) in `etc/skeleton/driver/main.go.tpl`.